### ISSUE TITLE
Using LLD

### ISSUE THREAD

**Issue by mbribeiro** [hengyunabc/zabbix-sender#14]
Hi, I am trying to send a Low Levewl Discovery message (data message not status). Is this possible using this library

**Comment by exic** [comment]
Yes, your JSON message has to contain everything. It's pretty generic.
